### ISSUE TITLE
Drug Discovery using Deep Learning Algorithms

### ISSUE THREAD

**Issue by suryaremanan** [ersilia-os/ersilia#90]
**Medicinal Literature**
<ol>
<li> Collect Data from various open sources (learn more about the Isaura Data Store)</li>
<li> Perform EDA </li>
<li> Dimensionality Reduction algorithms (like PCA and t-SNE)</li>
<li> visualize the chemicals using matplotlib, plotly, dash</li>
<li> Combine two or more chemicals to form new drugs and make sense out of it (this will require domain expertise) </li>
</ol>

**Models used**
<ul> 
<li>CNN</li>
<li> Softmax classifier</li>
<li>GANs</li>
</ul>

**Training**
After a lot of epochs, the model should predict the correct drug with a good accuracy

**Discussion**
Please provide your valuable feedback and hopefully we might end up discovering a novel drug.
Thank You
-Kumari Surya Remanan

**Comment by anamika-yadav99** [comment]
Hi @suryaremanan This is what the Ersilia is doing. It takes the data from scientists then train a model ,package it and deploy it.  Then it provides an API for the model which makes it easier for people in scientific community with limited resources and coding skills to use it. It actually even takes pre-trained models and 3rd party model and deploys it. So, if you have any particular dataset in mind you could go ahead work on it. I'm also very interested in application of ML in Biotechnology. If you do select a dataset do let me know if you want any help. I would love to contribute.

**Comment by deepthi1107** [comment]
Hello, @suryaremanan I am an outreachy applicant, Can you assign me for this issue, I would like to contribute.

**Comment by GemmaTuron** [comment]
Hello @suryaremanan,

Can you provide more information of what is the issue about? Do you want to deploy a particular ML model as @anamika-yadav99 is suggesting?
@deepthi1107, I am the mentor for this project, before assigning this issue I would like to understand better what is it referring to.

Please go to the Slack channel where we will now post how to continue contributing to this project

**Comment by deepthi1107** [comment]
sure @GemmaTuron

**Comment by GemmaTuron** [comment]
Hello @suryaremanan
Please provide some more information on what are you suggesting as a task, otherwise we will close this issue due to inactivity

**Comment by suryaremanan** [comment]
Hello, @miquelduranfrigola and @GemmaTuron I want to work on improving the existing models.

**Comment by GemmaTuron** [comment]
Hi @suryaremanan 

Thanks for your interest. we are currently not able to provide individual guidance on how to train new models. If you have already done models that you wish to submit to the Hub, please do so via the github issues or request a model via our website
